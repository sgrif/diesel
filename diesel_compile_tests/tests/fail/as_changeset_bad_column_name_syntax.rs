#[macro_use]
extern crate diesel;

table! {
    users {
        id -> Integer,
        name -> Text,
    }
}

#[derive(AsChangeset)]
#[diesel(table_name = users)]
struct User {
    #[diesel(column_name)]
    name: String,
}

fn main() {}
