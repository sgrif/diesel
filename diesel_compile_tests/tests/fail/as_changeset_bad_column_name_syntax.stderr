error: `column_name` is either unexpected or has bad format
  --> $DIR/as_changeset_bad_column_name_syntax.rs:14:14
   |
14 |     #[diesel(column_name)]
   |              ^^^^^^^^^^^
